distributions/kfdef/servicemesh/kfctl_openshift_v1.3.0.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: kubeflow
+  namespace: kubeflow
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/openshift-scc
+      name: openshift-scc
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/cert-manager
+      name: cert-manager
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/admission-webhook
+      name: admission-webhook
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/openshift-service-mesh
+      name: istio-managed
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/knative
+      name: knative
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/jupyter-web-app
+      name: jupyter-web-app
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/notebook-controller
+      name: notebook-controller
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/centraldashboard
+      name: centraldashboard
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/volumes-web-app
+      name: volumes-web-app
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/pytorch-job
+      name: pytorch-job
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/tf-job
+      name: tf-job
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/katib
+      name: katib
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed/application/kfp-tekton
+      name: kfp-tekton
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift/application/kfserving
+      name: kfserving
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: distributions/stacks/openshift-managed
+      name: kubeflow-apps
+  repos:
+    - name: manifests
+      uri: 'https://github.com/VaishnaviHire/manifests/archive/test_sm_istio.tar.gz'
+  version: v1.3.0
```

Comment:
@VaishnaviHire we might have to move this to KF 1.4 since it will be included in next ODH v1.1.2 release